manifests/bosh-manifest/deployments/010-bosh-template.yml

```diff
@@ -58,7 +58,6 @@ jobs:
         max_orphaned_age_in_days: 0
       cpi_job: (( grab cloud_provider.template.name ))
       user_management:
-        provider: local
```

Comment:
As commented in https://github.com/alphagov/paas-cf/pull/463/commits/7d19a23b26e6051e1914a91c3c73cdd1b9350e88#r77513225, setting this explicitly  makes it easier to understand.